src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -170,6 +172,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
     [OuterLoop]
     public static void SendTimeout_Zero_Throws_TimeoutException_Immediately()
     {
+        TimeSpan serviceOperationTimeout = TimeSpan.FromMilliseconds(10000);
         BasicHttpBinding binding = new BasicHttpBinding();
```

Comment:
If we expect this to timeout immediately, I recommend we reduce how long we ask the server to stall (5 seconds?) just so the server side isn't tied up so long.  Your call.